src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin: 0;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
```

Comment:
This is a good use of the `:not(:first-child)` pseudo-class to apply left margin spacing between star elements. However, ensure that adding margin only to one side (left or right) follows the design requirements and consistency in margin usage.